### PR TITLE
Meal route

### PR DESCRIPTION
**Pull request by stiehlrod**
This adds Objection (an ORM that works with Knex), the benefit of which is eager loading. Also added were the models and their relationships.
Package.json updates:
-Objection
-Pry JS
-Knex update

request: api/v1/meals

response:
```json
[
    {
        "id": 1,
        "name": "Breakfast",
        "foods": [
            {
                "name": "apple",
                "calories": 220
            },
            {
                "name": "scrambled eggs",
                "calories": 350
            },
            {
                "name": "banana",
                "calories": 150
            }
        ]
    },
    {
        "id": 2,
        "name": "Lunch",
        "foods": [
            {
                "name": "coke",
                "calories": 320
            },
            {
                "name": "hamburger",
                "calories": 780
            },
            {
                "name": "french fries",
                "calories": 350
            }
        ]
    },
    {
        "id": 3,
        "name": "Dinner",
        "foods": [
            {
                "name": "green beans",
                "calories": 350
            },
            {
                "name": "steak",
                "calories": 350
            },
            {
                "name": "potato",
                "calories": 350
            }
        ]
    },
    {
        "id": 4,
        "name": "Snack",
        "foods": [
            {
                "name": "yogurt",
                "calories": 350
            }
        ]
    }
]
```

### CODE REVIEW

File: routes/api/v1/foods.js [hunk 1]
```diff
@@ -16,7 +16,14 @@ router.get('/', function(req, res, next) {
       res.status(500).json({ error });
     });
 });
-
+// const Food = require('../../../models/food')
```
Comment (milevy1) [inline routes/api/v1/foods.js]: Why did you comment this out? Was it breaking things with it?

File: knexfile.js [hunk 2]
```diff
@@ -1,10 +1,22 @@
 // Update with your config settings.
+// const config = require('config');
```
Comment (milevy1) [inline knexfile.js]: Are these commented lines no longer needed?

File: knexfile.js [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+// const postgres = config.get('postgres');
+// require('dotenv').config({path: 'path_to_env_file'});
+// require('dotenv').config()
+// const pg = require('pg')
+// pg.defaults.ssl = true
 
 module.exports = {
-
+  client: 'pg',
+  // connection: process.env.DATABASE_URL,
```
Comment (milevy1) [inline knexfile.js]: Is this connection line no longer needed?

File: models/food.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+// const knex = require('knex')
```
Comment (milevy1) [inline models/food.js]: Is this line replaced with line 2 now?

**Comment by milevy1** [comment]
@stiehlrod I added a few comments.

Also, the spec asks for `id` to be shown for each food. Can that be easily included with Objection?

File: routes/api/v1/foods.js (hunk 1 above)
Comment (stiehlrod) [reply routes/api/v1/foods.js]: currently, it is not necessary. I think if we switch the foods routes to use Objection, then we may need it.

**Pull request merged**